app/managers.py

```diff
@@ -0,0 +1,40 @@
+from models import Actor
+
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name,)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        actors_format_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actors_format_cursor]
+
+    def update(self,
+               id_to_update: int,
+               new_first_name: str,
+               new_last_name: str) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table_name} SET (first_name, last_name)"
+            f" = (?, ?) WHERE id = ?",
+            (new_first_name, new_last_name, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_detele: int) -> None:
```

Comment:
There is a typo in the parameter name 'id_to_detele'. It should be corrected to 'id_to_delete' for consistency and clarity.